packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/quanticSmartSnippetAnswer.js

```diff
@@ -59,34 +59,59 @@ export default class QuanticSmartSnippetAnswer extends LightningElement {
     // eslint-disable-next-line @lwc/lwc/no-inner-html
     snippetAnswerElement.innerHTML = this?.answer;
     this.bindAnalyticsToSmartSnippetInlineLinks();
+    this.disableInvalidInlineLinks();
```

Comment:
I think this could be merged into something like:

```
this.handleLinksInSmartSnippets();

handleLinksInSmartSnippets() {
  this.inlineLinks.forEach((link) => {
    if(link?.href) {
      // Bind analytics
    } else {
      link.classList.add(disabledCSSClass);
    }
  }
}
```